<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event</class>
 <widget class="QWidget" name="Event">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Дата события</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="dayBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Неизвестно</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>21</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>22</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>23</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>24</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>25</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>26</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>27</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>28</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>29</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>30</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>31</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="monthBox">
             <property name="editable">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Неизвестно</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Январь</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Февраль</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Март</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Апрель</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Май</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Июнь</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Июль</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Август</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Сентябрь</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Октябрь</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ноябрь</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Декабрь</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yearEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>г.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Тематика</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="themeBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Краткое описание события</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="shortEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Полное описание события</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="fullEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Место события</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="placeBox">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="duplicatesEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Источник</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sourceBox">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Дополнительно</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="extraEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Фотография</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="uploadButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeImage">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nextImage">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="previousImage">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="currentPhoto">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Подтвердить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
